sdk/keyvault/azure-security-keyvault-certificates/perf-tests.yml

```
@@ -14,4 +14,5 @@ Tests:
 - Test: get-certificate
   Class: GetCertificate
   Arguments:
-  - --parallel 64
+    - --warmup 0 --duration 1
+    - --parallel 4
```

```suggestion
  - --parallel 4
```

NIT: I think you can indent this one level shallower.